lib/ts/recipe/session/components/themes/accessDeniedScreenTheme/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/jsx-no-literals */
+import React from "react";
+
+import ErrorRoundIcon from "../../../../../components/assets/errorRoundIcon";
+import { withOverride } from "../../../../../components/componentOverride/withOverride";
+import { hasFontDefined } from "../../../../../styles/styles";
+import SuperTokens from "../../../../../superTokens";
+import { useTranslation } from "../../../../../translation/translationContext";
+import { useUserContext } from "../../../../../usercontext";
+import BackButton from "../../library/backButton";
+import LogoutButton from "../../library/logoutButton";
+import { ThemeBase } from "../themeBase";
+
+import type { AccessDeniedThemeProps } from "../../../types";
+import type { FC } from "react";
+
+const AccessDeniedScreen: FC<AccessDeniedThemeProps> = (props) => {
+    const userContext = useUserContext();
+    const t = useTranslation();
+    const onLogout = async () => {
+        try {
+            await props.recipe.signOut({ userContext });
+        } catch (error) {
+        } finally {
+            await SuperTokens.getInstanceOrThrow().redirectToAuth({
+                show: "signin",
+                redirectBack: false,
+            });
+        }
+    };
+
+    const onBackButtonClicked = () => {
+        if (props.history.back !== undefined) {
+            return props.history.back();
+        }
+        return props.history(-1);
```

Comment:
This needs to handle 3 cases:
- react-router-dom>6
- react-router-dom 5
- no react-router-dom

Did you test this? I don't think this works:
- history is undefined without react-router-dom (I think this is what you tried to handle)
- react-router-dom 5 history has `goBack`